recipe/0001-support-ffmpeg-and-not-sox.patch

```diff
@@ -0,0 +1,50 @@
+From 749175a09a460821ae03fb5634d3ec9bf8dabbe0 Mon Sep 17 00:00:00 2001
+From: Jack Olivieri <jolivieri@anaconda.com>
+Date: Mon, 1 Jul 2024 12:28:54 +0200
+Subject: [PATCH 1/2] support ffmpeg and not sox
+
+---
+ CMakeLists.txt                   | 6 ++++--
+ tools/setup_helpers/extension.py | 4 +++-
+ 2 files changed, 7 insertions(+), 3 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f0195c87..a9457700 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -52,13 +52,15 @@ endif()
+ 
+ 
+ # Options
+-option(BUILD_SOX "Build libsox statically" ON)
++option(BUILD_SOX "Build libsox statically" OFF)
++# really making sure sox is disabled and isn't being built or linked
++SET(BUILD_SOX OFF CACHE BOOL "Build libsox into libtorchaudio")
+ option(BUILD_RIR "Enable RIR simulation" ON)
+ option(BUILD_RNNT "Enable RNN transducer" ON)
+ option(BUILD_ALIGN "Enable forced alignment" ON)
+ option(BUILD_CUDA_CTC_DECODER "Build CUCTC decoder" OFF)
+ option(BUILD_TORCHAUDIO_PYTHON_EXTENSION "Build Python extension" OFF)
+-option(USE_FFMPEG "Enable ffmpeg-based features" OFF)
```

Comment:
`option` means you can specify the option as a cmake flag in the build.sh/bld.bat rather than patching

Reply:
done